engine/dlib/src/dlib/intersection.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright 2020-2022 The Defold Foundation
+// Copyright 2014-2020 King
+// Copyright 2009-2014 Ragnar Svensson, Christian Murray
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+//
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#include <dmsdk/dlib/intersection.h>
+#include <stdint.h>
+
+namespace dmIntersection
```

Comment:
Maybe it's not necessary, but do we want any tests for this api?

Reply:
Yes, it's included in the other "real" PR: https://github.com/defold/defold/pull/6449